argus/client/sct/client.py

```diff
@@ -0,0 +1,156 @@
+from uuid import UUID
+import requests
+from argus.backend.util.enums import ResourceState, TestStatus
+from argus.client.base import ArgusClient
+
+
+class ArgusSCTClient(ArgusClient):
+    test_type = "scylla-cluster-tests"
+    schema_version: str | None = "v8"
+
+    class Routes(ArgusClient.Routes):
+        SUBMIT_PACKAGES = "/sct/$id/packages/submit"
+        SUBMIT_SCREENSHOTS = "/sct/$id/screenshots/submit"
+        CREATE_RESOURCE = "/sct/$id/resource/create"
+        TERMINATE_RESOURCE = "/sct/$id/resource/$name/terminate"
+        UPDATE_SHARDS_FOR_RESOURCE = "/sct/$id/resource/$name/shards"
+        SUBMIT_NEMESIS = "/sct/$id/nemesis/submit"
+        FINALIZE_NEMESIS = "/sct/$id/nemesis/finalize"
+        SUBMIT_EVENTS = "/sct/$id/events/submit"
+
+    def __init__(self, run_id: UUID, auth_token: str, base_url: str, api_version="v1") -> None:
+        super().__init__(auth_token, base_url, api_version)
+        self.run_id = run_id
+
+    def submit_sct_run(self, job_name: str, job_url: str, started_by: str, commit_id: str,
+                       runner_public_ip: str, runner_private_ip: str, sct_config: str) -> requests.Response:
```

Comment:
what is `sct_config` string? Do you mean `sct_config_path` to refer path of the config?

Reply:
Good catch, this is supposed to be dict and the intention that you pass whole `SCTConfiguration` class instance as a dict.

Reply:
Done